fe/spark-dpp/src/main/java/org/apache/doris/load/loadv2/dpp/ColumnParser.java

```diff
@@ -71,14 +71,25 @@ public static ColumnParser create(EtlJobConfig.EtlColumn etlColumn) throws Spark
         }
     }
 
-    public abstract boolean parse(String value);
+    public abstract boolean parse(Object value);
+}
+
+class BinaryParser extends ColumnParser{
+    @Override
+    public boolean parse(Object value) {
+        try{
+            return value instanceof byte[];
+        }catch (NumberFormatException e) {
+            return false;
+        }
+    }
 }
 
 class TinyIntParser extends ColumnParser {
     @Override
-    public boolean parse(String value) {
+    public boolean parse(Object value) {
         try {
-            Byte.parseByte(value);
+            Byte.parseByte(value.toString());
```

Comment:
If value is null, then NPE happens here.

Reply:
NPE will also occur in the previous code, and NPE has been try catch.

Reply:
NumberFormatException is included but NPE will not catched in this situation.